index.html

```diff
@@ -31,7 +31,7 @@ <h2>Release Calendar</h2>
 
 		<p><i>* Disclaimer 1: Out of all test releases, only release deployed to Kovan network contains all necessary configuration required for full testing.</i></p>
 		<p><i>* Disclaimer 2: Releases that contain <b>active</b> in the URL are those that have been deployed, executed and in use.</i></p>
-		<p><i>* Disclaimer 3: Releases that contain <b>number</b> (e.g. 1.1.0) in the URL might be deployed, but not yet in use.</i></p>
+		<p><i>* Disclaimer 3: Releases that contain <b>number</b> (e.g. 1.1.0) in the URL are deployed, but might not yet be in use.</i></p>
```

Comment:
This is grammatically correct, albeit slightly awkward.  IMO, we could eliminate the word `yet`.  This would better handle the case where we deploy contracts, but the vote fails.